jenkins/pipelines/cd/nightly/release-all-images-by-branch.groovy

```diff
@@ -740,6 +740,7 @@ try {
     }
 } catch (Exception e) {
     currentBuild.result = "FAILURE"
+    throw e
 } finally {
```

Comment:
no need to add catch block, Only need finally block.
Ref: https://www.tutorialspoint.com/groovy/groovy_exception_handling.htm